### PR TITLE
Updated the wseb version to 1.0 to enable testing against current implementation

### PR DESCRIPTION
**Pull request by pkhanal**
Also fixed the strictness in the scripts those were expecting 'read closed' immediately after response with status code 400

### CODE REVIEW

File: specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt [hunk 1]
```diff
@@ -36,22 +36,20 @@ write close
 read status "200" /.+/
 read header "Content-Type" "application/octet-stream"
 read header "Connection" "close"
+read notify DOWNSTREAM_OPENED
 
-read [0x80 0x82 0x00]
-read [0..256]
-read notify DATA_RECEIVED
```
Comment (cmebarrow) [inline specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt]: Why did you remove this? Just because it's unnecessary? (which is a fine reason, just wanted to know if there was any other)

File: specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt [hunk 2]
```diff
@@ -7,14 +7,14 @@ write method "POST"
...
+write header "X-WebSocket-Version" "wseb-1.0"
 write header "X-Sequence-No" ${wse:asString(sequence)}
 write close
 
 read status "201" /.+/
 read version "HTTP/1.1"
-read header "X-WebSocket-Version" "wseb-1.1"
-read header "Content-Type" "text/plain;charset=utf-8"
+read header "X-WebSocket-Version" "wseb-1.0"
+read header "Content-Type" /text\/plain;charset=(?i)utf-8/
```
Comment (cmebarrow) [inline specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt]: Why do we need the "?i"? Does the server add a space?

**Comment by cmebarrow** [comment]
@pkhanal LGTM, just a couple of minor questions (see my comments). Do you think we should add a negative test for sequencing (i.e. out of order request)? Spec says that should result in 400 bad request.

File: specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt (hunk 1 above)
Comment (pkhanal) [inline specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt]: I wanted to make the test as much tight as possible without losing the purpose of the test.

File: specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt (hunk 2 above)
Comment (pkhanal) [inline specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt]: It's just to make the charset value case insensitive. I have seen cases where server would return "text/plain;charset=UTF-8". Semantically it is equivalent to "text/plain;charset=utf-8". It may not be robust for all combinations but we can fix that as we come across those scenarios.

**Comment by pkhanal** [comment]
@cmebarrow, we already have negative tests for sequencing. They are available for [upstream requests](https://github.com/k3po/k3po/tree/develop/specification/wse/src/main/scripts/org/kaazing/specification/wse/upstream/request.out.of.order) and [downstream](https://github.com/k3po/k3po/tree/develop/specification/wse/src/main/scripts/org/kaazing/specification/wse/downstream/request.out.of.order) requests.

**Comment by cmebarrow** [comment]
@pkhanal OK, thanks for your answers. Could you look into the build failure? Test MessageFormatIT.inboundShouldAcceptHeaders failed on openJdk. Could be a sporadic failure. If so please file a bug and rebuild.

**Comment by pkhanal** [comment]
@cmebarrow, there is already an [issue](https://github.com/k3po/k3po/issues/187) filed for the failure you pointed.

**Comment by cmebarrow** [comment]
OK, merging
